comfy_cli/cloud/command.py

```
@@ -0,0 +1,364 @@
+        str | None,
+        typer.Argument(
+            help="Base URL like 'https://fe-pr-12159.testenvs.comfy.org'. Pass '' or omit and use --clear to reset.",
+        ),
+    ] = None,
+    clear: Annotated[
+        bool,
+        typer.Option("--clear", help="Clear the persisted base URL (falls back to env var, then default)."),
+    ] = False,
+):
+    from comfy_cli.cloud import CONFIG_KEY_BASE_URL
+    from comfy_cli.config_manager import ConfigManager
```

Pylint (4.0.5)

[warning] Redefining name 'ConfigManager' from outer scope (line 28)

(W0621)